hpsklearn/components.py

```diff
@@ -1325,7 +1326,7 @@ def _xgboost_hp_space(
                     if reg_lambda is None else reg_lambda),
         scale_pos_weight=scale_pos_weight,
         base_score=base_score,
-        seed=_random_state(name_func('rstate'), random_state)
+        seed=_random_state(name_func('rstate'), random_state=random_state, n_jobs=n_jobs)
```

Comment:
n_jobs=n_jobs needs to be outside the parentheses here. Where it is now it is actually incorrectly being passed to _random_state

Reply:
You're right! Sorry about that.